src/coreclr/vm/util.cpp

```diff
@@ -177,10 +180,14 @@ void PrintToStdErrA(const char *pszString) {
     }
     CONTRACTL_END
 
-    HANDLE  Handle = GetStdHandle(STD_ERROR_HANDLE);
-
+#if defined(TARGET_ANDROID)
+    // TODO: make priority configurable?
```

Comment:
This is used to print fatal errors right before the process aborts. This should be something that gets always printed to the log. I do not think there is any situation where one would not want to see it in the log.

Reply:
So what you're saying is we do both, right?

Reply:
Actually the ifdef'd out branch is printing to stderr so I'm not sure what is desired here.

Reply:
I was commenting on the priority TODO. It should be hardcoded to highest possible priority.

stdout and stderr are useless on Android.